src/main/webapp/pages/template/main-navigation.html

```diff
@@ -212,15 +212,15 @@
                         </ul>
                     </li>
                     <li class="dropdown">
-                        <a href="#" class="dropdown-toggle nav-gravatar-wrapper" data-toggle="dropdown" role="button" aria-haspopup="true"
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
                            aria-expanded="false">
-                            <img class="nav-gravatar" th:src="${session.gravatar}" alt="User Icon"/>
+                            <i class="far fa-user nav-icon"></i>
```

Comment:
Thoughts on using a circle style icon instead so the style stays a bit more familiar until we get the new topbar done?  If we used something like this we can get by without a whole bunch of docs changes.  https://fontawesome.com/icons/user-circle?style=solid

Reply:
Updated icon in 70f5272a1e2d1e50b3594b0ed453dd24905b7c4d